### ISSUE TITLE
Current title does not communicate resource purpose and scope

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wcag-act-rules#101]
[Ref review comment](https://github.com/w3c/wcag-act-rules/pull/95/files#r865692657)

Current title is "Accessibility Test Tools and Methodologies"
URL is /standards-guidelines/act/implementations/

The title does not communicate well the purpose of the pages. 

What are we saying about "Accessibility Tools and Methodologies"? Are we showing a list? Are we showing how many rules each of them has implemented? Are we providing definitions for different types of test tools and methodologies?
These are things we should be clear on to look for a better title for this page. Just by saying "Accessibility Test Tools and 
Methodologies" we are not fully communicating what's on the page.

I remember discussions around the word "metrics" back in the day. Why this has been ditched?

**Comment by WilcoFiers** [comment]
Noted, but let's have the conversation on PR #95. Lets reserve issues for things that are live, and put comments about PRs on that PR.
